code/modules/mob/living/silicon/robot/robot_modules.dm

```diff
@@ -221,6 +219,12 @@ var/global/list/robot_modules = list(
 	src.modules += new /obj/item/device/reagent_scanner/adv(src)
 	src.modules += new /obj/item/autopsy_scanner(src)
 	src.modules += new /obj/item/device/breath_analyzer(src)
+	src.modules += new /obj/item/taperoll/engineering(src) // To enable 'borgs to telegraph danger visually.
+	src.modules += new /obj/item/inflatable_dispenser(src) // To enable 'borgs to protect Crew from danger in direct hazards.
+	src.modules += new /obj/item/device/gps(src) // For being located while disabled and coordinating with life sensor consoles.
+	src.modules += new /obj/item/extinguisher/mini(src) // For navigating space and/or low grav, and just being useful.
+	src.modules += new /obj/item/device/flash(src) // Non-lethal tool that prevents any 'borg from going lethal on Crew so long as it's an option according to laws.
+	src.modules += new /obj/item/crowbar/robotic(src) // Base crowbar that all 'borgs should have access to.
```

Comment:
I feel like these comments are largely useless

Reply:
They kinda are and were more meant just to explain the thought process behind adding them to each 'borg module.

Reply:
This I hope is resolved, reopen if not. (I'll still remove them if you'd like me to)